### ISSUE TITLE
Virus

### ISSUE THREAD

**Issue by DamianTC1903** [TeamNewPipe/NewPipe#8689]
### Checklist

- [X] I am able to reproduce the bug with the [latest version](https://github.com/TeamNewPipe/NewPipe/releases/latest).
- [X] I made sure that there are *no existing issues* - [open](https://github.com/TeamNewPipe/NewPipe/issues) or [closed](https://github.com/TeamNewPipe/NewPipe/issues?q=is%3Aissue+is%3Aclosed) - which I could contribute my information to.
- [X] I have taken the time to fill in all the required details. I understand that the bug report will be dismissed otherwise.
- [X] This issue contains only one bug.
- [X] I have read and understood the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/dev/.github/CONTRIBUTING.md).

### Affected version

All

### Steps to reproduce the bug

https://www.virustotal.com/gui/file/9a589053843ec96daf364ab648a1f58c364173e25d752dde36625ec1868e1c55/detection

### Expected behavior

contains viruses, takes advantage of users, report to github

### Actual behavior

contains viruses, takes advantage of users, report to github

### Screenshots/Screen recordings

![Screenshot_20220724-131919_Chrome](https://user-images.githubusercontent.com/74327566/180660778-14d62d5f-65d0-4957-8edb-7fb9099138d2.png)

**Comment by mhmdanas** [comment]
Thank you for the report.

I'm inclined to believe that this is a false positive. I uploaded the NewPipe 0.23.1 APK built by F-Droid to VirusTotal, and it gave back no results. Just to be sure, I also uploaded the NewPipe 0.23.1 APK (and also the 0.23.0 APK) from GitHub releases, and I got back the same result as in this issue. I then proceeded to diff the two APKs, and here are the results:

```
~/D/newpipe-apks> diff -r ours f-droid/
Only in f-droid/META-INF: EE8807D2.RSA
Only in f-droid/META-INF: EE8807D2.SF
Only in ours/META-INF: SCHABI_K.RSA
Only in ours/META-INF: SCHABI_K.SF
```

Both files are related to the signing of the APK, and have nothing to do with the code run by the app. Them being different is totally expected.

In light of the above, and given that only one of the "security vendors" detected this, I think this is a false positive. Can anybody provide evidence that the GitHub APKs do contain malware/trojan horses?
